deepspeed/launcher/multinode_runner.py

```diff
@@ -73,8 +73,8 @@ def name(self):
 
     def get_cmd(self, environment, active_resources):
         environment['PDSH_RCMD_TYPE'] = 'ssh'
-        if self.args.ssh_port is not None:  # only specify ssh port if it is specified
-            environment["PDSH_SSH_ARGS_APPEND"] += f" -p {self.args.ssh_port}"
+        if self.args.ssh_port is not None:   # only specify ssh port if it is specified
```

Comment:
FYI @Lzhang-hub - can you remove this added space on line 76?  Its causing the formatting checks to fail.

Reply:
ok